### DISCUSSION TITLE
How to create multiple column layout

### DISCUSSION THREAD

**Discussion by joeyseisma** [reflex-dev/reflex#756] [Q&A]
Hi,

Initially, I'd like to create 2 columns, left column the links, right column result of the link.

I'd like it resizable or collapsible too.

How to do? 

TIA!

**Comment by ArnauldFilee** [comment]
You can just use the pc.vstack and pc.hstack. 
For exemple :
```
pc.hstack(
    pc.vstack(
        pc.h1("Link"),
    ),
    pc.vstack(
        pc.h2("result of the link"),
    ),
),
```
